src/sql/workbench/contrib/dataExplorer/browser/dataExplorer.contribution.ts

```diff
@@ -42,6 +42,16 @@ configurationRegistry.registerConfiguration({
 		'datasource.connectionGroups': {
 			'description': localize('datasource.connectionGroups', "data source groups"),
 			'type': 'array'
+		},
+		'datasource.connectionSort': {
+			'type': 'string',
+			'enum': ['byTimeAdded', 'byTitleAlphabetically'],
+			'enumDescriptions': [
+				localize('connectionSort.byTimeAdded', 'Saved connections are sorted by the time they were added.'),
+				localize('connectionSort.byTitleAlphabetically', 'Saved connections are sorted by their titles alphabetically.')
```

Comment:
This should be updated to correspond to the new enum value as well

Reply:
updated